blackbox/docs/sql/general/value-expressions.rst

```diff
@@ -123,7 +123,8 @@ Function Call
 A function is declared by its name followed by its arguments enclosed in
 parentheses::
 
-    function_name([expression [, expression ... ]]) [ OVER([ORDER BY expression] [, ...]) ]
+    function_name([expression [, expression ... ]])
+      [OVER( [PARTITION BY expression] [, ...] [ORDER BY expression] [, ...]) ]
```

Comment:
I think the `[, ... ]` to indicate that the expression can be a list should go inside the optional PARTITION BY or ORDER BY blocks like this:

```suggestion
      [OVER( [PARTITION BY expression [, ...] ] [ORDER BY expression [, ...] ]) ]
```